alis.sh

```diff
@@ -1302,6 +1305,11 @@ function desktop_environment_lxde() {
     arch-chroot /mnt systemctl enable lxdm.service
 }
 
+function desktop_environment_i3wm() {
+    pacman_install "i3 dmenu rxvt-unicode lightdm lightdm-gtk-greeter xorg-server"
```

Comment:
i3 package group contains i3-wm and i3-gaps and in the Arch Linux wiki there is the following message "Note: i3-wm conflicts with i3-gaps (a fork of i3 with gaps and other features) and by default i3-gaps will be installed."

Would be possible to allow the user to choose between i3-wm and i3-gaps when installing i3 package group? If it is necessary using in the DESKTOP_ENVIRONMENT these two different values i3wm and i3wm-gap to execute a different pacman_install command.